v3/src/utilities/view-utils.ts

```diff
@@ -93,3 +93,10 @@ export const getPositionOfNewComponent = (iViewRect: {width: number, height: num
   const nLoc = findEmptyLocationForRect()
   return nLoc
 }
+
+export const isWithinBounds = (dimension: number, rect?: DOMRect) => {
+  if (!rect) return
```

Comment:
Should probably return `false` here unless you're expecting clients to care about the distinction between `undefined` and `false`.